extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/QuarkusPathLocationScanner.java

```diff
@@ -42,7 +42,7 @@ public QuarkusPathLocationScanner(Collection<Location> locations) {
                 scannedResources.add(new ClassPathResource(null, migrationFile, classLoader, StandardCharsets.UTF_8));
             } else if (migrationFile.startsWith(Location.FILESYSTEM_PREFIX)) {
                 if (fileSystemScanner == null) {
-                    fileSystemScanner = new FileSystemScanner(StandardCharsets.UTF_8, false);
+                    fileSystemScanner = new FileSystemScanner(StandardCharsets.UTF_8, false, false, false);
```

Comment:
We will probably want to add configuration properties for these but I'll let our community add them if they need them.

Reply:
👍🏼